### ISSUE TITLE
Report duplicate blocks

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#149]
A feature in SonarQube is to report duplicate selectors. It would be interesting to take this one step further and report when complete blocks are duplicated.

```css
.my-class {
  color: red;
  background: white;
}

/*  ... lots of code ... */

.my-class {
  color: red;
  background: white;
}
```

There's a couple of caveats here:

- That same block in a `@media` query should not be reported
